src/features/ui/DropdownIconLabel/DropdownIconLabel.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+
+import styles from './DropdownIconLabel.module.scss';
+
+export type DropdownIconLabelProps = {
+    icon: JSX.Element,
+    label: string;
+};
+
+export const DropdownIconLabel: FC<DropdownIconLabelProps> = ({
+	icon,
+    label
+}) => (
+    <div className={styles.Container}>
+        {icon}
+        <div className={styles.Label}>
+            <span>{label}</span>
+        </div>
+    </div>
+);
```

Comment:
Can we move this functionality to zUI?

Reply:
Oh nice, yes we should move this over as we'll be re-using this plenty of times

Reply:
Yep, happy to move! Are you okay for me to address as a seperate piece? Stops this one getting blocked by a zUI update.